services/indexes/avax/writer.go

```diff
@@ -12,12 +12,12 @@ import (
 
 	"github.com/ava-labs/avalanchego/ids"
 	"github.com/ava-labs/avalanchego/utils/crypto"
-	"github.com/ava-labs/avalanchego/utils/formatting"
+	formatting "github.com/ava-labs/avalanchego/utils/formatting/address"
```

Comment:
lol this rename is mad lazy - can we just update the references?

Reply:
sure, happy to update